cmd/otelcol/config/collector/config.d.linux/receivers/mongodb.discovery.yaml

```diff
@@ -21,7 +21,6 @@
 #         insecure: false
 #       hosts:
 #         - endpoint: '`endpoint`'
-#           transport: tcp
```

Comment:
This is no longer needed for discovery due to upstream change and transport key is no longer recognized either.

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/mongodbreceiver/README.md#example-configuration